### ISSUE TITLE
In the browser Debugger tab there is an assets folder inside the assets folder

### ISSUE THREAD

**Issue by KroneckerDeIta** [ember-cli/ember-cli#9713]
We've recently upgraded from ember-cli 3.20.2 to 3.28.4 and I can see this issue in the browser after we have deployed, specifically it looks like this:

```
Main Thread
  <hostname>
    <project name>
      assets
        assets
          ui-a34...59.js
          vendor-b19...62.js
        ui-a34...59.js
        vendor-b19...62.js
      ...
```

The assets seem to be repeated in an inner assets folder. But when I click the the inner assets files it says:

```
Error while fetching an original source: request failed with status 404
Source URL: <unknown>
```

and there are warning messages in the console as not able to retrieve source.

When I actually look in the dist directory in a terminal everything looks fine. There isn't an inner assets directory inside the assets directory.

Issue is seen in both latest versions of Firefox and Chrome.

The application seems to run fine otherwise.

What package is responsible for populating the information about the sources? If I knew that I could change the version and see if that makes a difference.

**Comment by KroneckerDeIta** [comment]
In the app/index.html file we don't use {[rootURL}}, we just specify the relative path to the us.js and vendor.js files. Could that be causing an issue?

**Comment by KroneckerDeIta** [comment]
So it looks like it is an issue with ember-cli-uglify setting I have in the ember-cli-build.js file. It looks like ember-cli-update has removed ember-cli-uglify as part of the update, but ember-cli-uglify is still having an effect as part of the build (which seems weird). When I add ember-cli-uglify 3.0.0 back to the package.json the build fails with:

```
Passing `async` property inside `options` is deprecated.
```

so I guess the package was removed for a reason? Am I allowed to still set ember-cli-uglify configuration? Am I setting things incorrectly? This is how I am doing it in ember-cli-build.js:

```
        'ember-cli-uglify': {
            exclude: [
                'file1.js'
            ]
        }
```

It is not essential we stop certain files from being uglified, but would be good if we could still do that.
